crates/rolldown/src/utils/chunk/render_chunk_exports.rs

```diff
@@ -23,19 +23,30 @@ pub fn render_chunk_exports(
   match options.format {
     OutputFormat::Esm => {
       let mut s = String::new();
+      let mut export_default_count = 0;
       let rendered_items = export_items
         .into_iter()
         .map(|(exported_name, export_ref)| {
           let canonical_ref = link_output.symbols.par_canonical_ref_for(export_ref);
           let symbol = link_output.symbols.get(canonical_ref);
-          let canonical_name = &chunk.canonical_names[&canonical_ref];
```

Comment:
should use `ns_alias.namesapce_alias_ref` when the related symbol has `namespace_alias`